doc/source/notebooks/models/load_model.ipynb

```diff
@@ -0,0 +1,356 @@
+       "          (output_proj): Linear(input_dim=2048, output_dim=2048, bias=False, init_fn=init_output_projection)\n",
+       "        )\n",
+       "        (self_attn_norm): None\n",
+       "        (self_attn_dropout): None\n",
+       "        (self_attn_residual): StandardResidualConnect()\n",
+       "        (encoder_decoder_attn): None\n",
+       "        (encoder_decoder_attn_dropout): None\n",
+       "        (encoder_decoder_attn_residual): None\n",
+       "        (encoder_decoder_attn_layer_norm): None\n",
+       "        (ffn_layer_norm): RMSNorm(normalized_shape=(2048,), eps=1E-05, elementwise_affine=True)\n",
+       "        (ffn): GLUFeedForwardNetwork(\n",
+       "          model_dim=2048, inner_dim_scale=0.666667, inner_dim_to_multiple=256\n",
+       "          (gate_proj): Linear(input_dim=2048, output_dim=5632, bias=False)\n",
+       "          (gate_activation): SiLU()\n",
+       "          (inner_proj): Linear(input_dim=2048, output_dim=5632, bias=False)\n",
+       "          (inner_dropout): Dropout(p=0.1, inplace=False)\n",
+       "          (output_proj): Linear(input_dim=5632, output_dim=2048, bias=False)\n",
+       "        )\n",
+       "        (ffn_dropout): None\n",
+       "        (ffn_residual): StandardResidualConnect()\n",
+       "      )\n",
+       "    )\n",
+       "    (layer_norm): RMSNorm(normalized_shape=(2048,), eps=1E-05, elementwise_affine=True)\n",
+       "    (dropout): Dropout(p=0.1, inplace=False)\n",
+       "  )\n",
+       "  (final_proj): Linear(input_dim=2048, output_dim=32000, bias=False, init_fn=init_final_projection)\n",
+       ")"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from fairseq2.models.llama import LLaMAConfig, create_llama_model\n",
+    "from fairseq2.data import VocabularyInfo\n",
+    "\n",
+    "custom_config = LLaMAConfig(\n",
+    "    model_dim=2048,                    # Model dimension\n",
+    "    max_seq_len=4096,                  # Maximum sequence length\n",
+    "    vocab_info=VocabularyInfo(\n",
+    "        size=32000,                    # Vocabulary size\n",
+    "        unk_idx=0,                     # Unknown index\n",
+    "        bos_idx=1,                     # Beginning of sequence index\n",
+    "        eos_idx=2,                     # End of sequence index\n",
+    "        pad_idx=None                   # Padding index\n",
+    "    ),\n",
+    "    num_layers=16,                     # Number of transformer layers\n",
+    "    num_attn_heads=32,                 # Number of attention heads\n",
+    "    num_key_value_heads=8,             # Number of key/value heads\n",
+    "    ffn_inner_dim=2048 * 4,            # FFN inner dimension\n",
+    "    dropout_p=0.1                      # Dropout probability\n",
+    ")\n",
+    "\n",
+    "model = create_llama_model(custom_config)\n",
```

Comment:
nit : maybe add a comment that this will init a model with some random weights.
I would show also device and dtype args that are important here